libogc/exception.c

```diff
@@ -240,17 +273,19 @@ static void waitForReload(void)
 
 //just implement core for unrecoverable exceptions.
 void c_default_exceptionhandler(frame_context *pCtx)
-{
+{	
 	const u16 console_height = 680;
 	const u16 console_width = 574;
 
+	lookup_xfb = VIDEO_GetNextFramebuffer();
+	VIDEO_WaitVSync();
 	GX_AbortFrame();
 	VIDEO_SetFramebuffer(exception_xfb);
 	__VIClearFramebuffer(exception_xfb, console_height * console_width * VI_DISPLAY_PIX_SZ, COLOR_BLACK);
-	__console_init(exception_xfb, 20, 20, console_height-40, console_width-40, 1280);
+	__console_init(exception_xfb, 16, 32, console_height-16, console_width-32, 1280);
```

Comment:
why the change in console size? also, it should be twice the offset to both sides. the 20 was to offset overscan on some screens (meaning its 40 for both sides of the screen)